app/database.py

```diff
@@ -20,6 +20,7 @@ class UserProfile:
     priority: MessagePriority = MessagePriority.BOTTOM
     coins: int = 0
     cps: float = 0.1
+    user: discord.User
```

Comment:
IMO either use the user id or don't include this: I left a separate comment on the get every user DB method about not including this.